local/modules/HookAnalytics/AdminIncludes/module_configuration.html

```diff
@@ -25,11 +25,11 @@
             <div class="col-md-12">
 
             {form name="hookanalytics.configuration.form"}
-                <form method="POST" id="hookanalytics-form" action="{url path="/admin/module/hookanalytics/save"}" {form_enctype form=$form} class="clearfix">
+                <form method="POST" id="hookanalytics-form" action="{url path="/admin/module/hookanalytics/save"}" {form_enctypeNOW()} class="clearfix">

```

Comment:
what is this smarty function ```form_enctypeNOW()``` ? I don't find its definition in any smarty plugin.